### PR TITLE
add blog post about v2.4.3

### PR DESCRIPTION
**Pull request by kadel**
/kind documentation

### CODE REVIEW

**Comment by feloy** [comment]
Kubernetes Tests finished successfully.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-kubernetes-tests-158.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-kubernetes-tests-158.html)

**Comment by feloy** [comment]
Unit Tests finished successfully.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-unit-tests-171.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-unit-tests-171.html)

**Comment by feloy** [comment]
OpenShift Tests finished with errors.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-openshift-tests-187.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-openshift-tests-187.html)

File: docs/website/blog/2021-12-14-odo-v2.4.3.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+$ odo create
+$ odo deploy
+```
+[![asciicast](https://asciinema.org/a/NAR0IDRqdmEcHWSD32Ebs8XtW.svg)](https://asciinema.org/a/NAR0IDRqdmEcHWSD32Ebs8XtW)
+
+#### Know issues wth `odo deploy`
+
+- Resources deployed using `odo deploy` can't be deleted using odo [#5246](https://github.com/redhat-developer/odo/issues/5246)
+- `odo deploy` doesn't work without running `odo create` or `odo push` first. [#5289](https://github.com/redhat-developer/odo/issues/5289)
+- There is no odo command that would show if `odo deploy` was executed or not and what it is its status [#5248](https://github.com/redhat-developer/odo/issues/5248)
```
Comment (feloy) [inline docs/website/blog/2021-12-14-odo-v2.4.3.md:38]:
```suggestion
- There is no odo command that would show if `odo deploy` was executed or not and what is its status [#5248](https://github.com/redhat-developer/odo/issues/5248)
```

**Review by valaparthvi — CHANGES_REQUESTED** [review]

File: docs/website/package-lock.json [hunk 2]
```diff
@@ -1,11114 +0,0 @@
-{
```
Comment (valaparthvi) [inline docs/website/package-lock.json]: Any reason for deleting this file?

Comment (kadel) [reply docs/website/package-lock.json:1]: ah, that was a mistake :-/ I'll remove this from the commit

**Comment by feloy** [comment]
Kubernetes Tests finished successfully.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-kubernetes-tests-166.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-kubernetes-tests-166.html)

**Comment by feloy** [comment]
Unit Tests finished successfully.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-unit-tests-179.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-unit-tests-179.html)

**Comment by feloy** [comment]
OpenShift Tests finished successfully.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-openshift-tests-195.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-openshift-tests-195.html)

**Comment by feloy** [comment]
Kubernetes Tests finished successfully.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-kubernetes-tests-167.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-kubernetes-tests-167.html)

**Comment by feloy** [comment]
Unit Tests finished successfully.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-unit-tests-180.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-unit-tests-180.html)

**Comment by feloy** [comment]
OpenShift Tests finished successfully.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-openshift-tests-196.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-openshift-tests-196.html)

**Comment by feloy** [comment]
Kubernetes Tests finished successfully.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-kubernetes-tests-168.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-kubernetes-tests-168.html)

**Comment by feloy** [comment]
Unit Tests finished successfully.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-unit-tests-181.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-unit-tests-181.html)

**Comment by feloy** [comment]
OpenShift Tests finished with errors.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-openshift-tests-197.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-openshift-tests-197.html)

File: docs/website/package-lock.json (hunk 2 above)
Comment (kadel) [reply docs/website/package-lock.json]: fixed

**Comment by feloy** [comment]
Kubernetes Tests finished successfully.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-kubernetes-tests-176.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-kubernetes-tests-176.html)

**Comment by feloy** [comment]
Unit Tests finished successfully.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-unit-tests-189.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-unit-tests-189.html)

**Comment by feloy** [comment]
OpenShift Tests finished successfully.
View logs: [TXT](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-openshift-tests-205.txt) [HTML](https://s3.eu-de.cloud-object-storage.appdomain.cloud/odo-tests-openshift-logs/pr-5290-openshift-tests-205.html)

**Pull request merged**